File: center/src/main/java/site/zido/center/dto/UserWithInfoDTO.java
package site.zido.center.dto;

import site.zido.entity.BankCard;

import java.util.List;

/**
 * description
 * <p>fill description</p>
 *
 * @author zido
 * @since 2017/6/9 0009
 */
public class UserWithInfoDTO {
    private String nickname;
    private List<BankCard> bankCards;
    private String CareerName;

    public String getNickname() {
        return nickname;
    }

    public UserWithInfoDTO setNickname(String nickname) {
        this.nickname = nickname;
        return this;
    }

    public List<BankCard> getBankCards() {
        return bankCards;
    }

    public UserWithInfoDTO setBankCards(List<BankCard> bankCards) {
        this.bankCards = bankCards;
        return this;
    }

    public String getCareerName() {
        return CareerName;
    }

    public UserWithInfoDTO setCareerName(String careerName) {
        CareerName = careerName;
        return this;
    }
}

File: center/src/main/java/site/zido/center/dto/UserWithInfoDTO.kt
 package site.zido.center.dto

 import site.zido.entity.BankCard
 import java.util.*
 import kotlin.collections.ArrayList

 /**
 * Created by CDDC on 2017/6/1.
 */
class UserWithInfoDTO {
    /**
     * 用户昵称
     */
    var nickname: String = ""


    var bankCards:List<BankCard>? = ArrayList()

     /**
      * 职业
      */
     var CareerName:String = ""
 }